addons/account/views/account_payment_term_views.xml

```diff
@@ -77,10 +77,10 @@
                                     Example:
                                     <field name="example_amount" class="oe_inline"/>
                                     on
-                                    <field name="example_date" class="fw-bold oe_inline"/>
+                                    <field name="example_date" class="fw-bold border-bottom oe_inline" style="width: 20% !important;"/>
```

Comment:
We try to avoid putting style attribute on field because style is most powerful attribute in terms of css. I mean that style will override the class, so we always prefer to put css class than putting inline styling :smile: